core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -170,3 +170,5 @@ workflows:
     platform: gcp
   openshift-upgrade-gcp-ovn-rt:
     platform: gcp
+  hypershift-aws-conformance:
+    platform: hypershift
```

Comment:
`hypershift` is already a "supported" platform for the cluster-bot.  So, ideally, this particular workflow should be used to correct the existing (broken) workflow that's currently run when users run `launch 4.11 hypershift`:
https://github.com/openshift/release/blob/33e6f1fe453db8d987d576d1056fce75aaa9aaac/ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml#L2442-L2468

Reply:
So to make sure I understand: 
is this hardcoded UNRESOLVED_CONFIG config what would run by default when users run `launch 4.11 hypershift`?
and when using `workflow-launch hypershift-aws-conformance` UNRESOLVED_CONFIG gets overridden with the workflow passed?

Reply:
Yes.  Folks tend to use the `workflow-launch` as a way to test out new workflows and not necessarily expose them to the world (i.e. you have to go searching for them to know they exist).  Once they have been proven, they typically get added as actual `launch` command platform/variants and removed from the workflow configuration all together.  Since the `hypershift` entry point already exists, then you can take full advantage of it.

Reply:
Updated the PR to default launch to this workflow. I'm keeping workflow-launch with the same so I can experiment  with that command after merging to follow up with upcoming workflows.